src/udisks2blockdevices.cpp

```diff
@@ -318,9 +318,23 @@ void BlockDevices::updateFormattingState(Block *block)
 
 void BlockDevices::dumpBlocks() const
 {
+    if (!m_activeBlockDevices.isEmpty())
+        qCInfo(lcMemoryCardLog) << "======== Active block devices";
```

Comment:
I wonder if this should also give you a count. Similarly for `m_blockDevices` below.

Reply:
Will add count.